frank-flow/src/frontend/src/app/shared/services/flow-namespace.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { CurrentFileService } from './current-file.service';
+import { File } from '../../shared/models/file.model';
+import { FlowStructure } from '../models/flow-structure.model';
+import { MonacoEditorComponent } from 'src/app/editor/monaco-editor/monaco-editor.component';
+import { FlowStructureService } from './flow-structure.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FlowNamespaceService {
+  public monacoEditorComponent?: MonacoEditorComponent;
+  private currentFile!: File;
+  private flowStructure!: FlowStructure;
+
+  constructor(
+    private currentFileService: CurrentFileService,
+    private flowStructureService: FlowStructureService
+  ) {
+    this.getCurrentFile();
+  }
+
+  getCurrentFile(): void {
+    this.currentFileService.currentFileObservable.subscribe({
+      next: (currentFile: File): void => {
+        this.currentFile = currentFile;
+        if (currentFile.flowStructure) {
+          this.flowStructure = currentFile.flowStructure;
+        }
+      },
+    });
+  }
+
+  handleNameSpace() {
+    if (!this.checkNameSpace()) {
+      return;
+    }
+    this.setNameSpace();
+  }
+
+  checkNameSpace(): boolean {
+    if (this.flowStructure.configuration !== undefined) {
+      for (const [attributeKey] of Object.entries(
+        this.flowStructure.configuration?.attributes
+      )) {
+        if (attributeKey === 'xmlns:flow') {
+          return false;
+        }
+      }
+    }
+    return true;
```

Comment:
In all fairness, there are some things wrong with this code, and you might overlook how simple the real solution can be: 
```suggestion
    return !this.flowStructure.configuration?.attributes['xmlns:flow']
```

This code is a translation of yours. But even than, I think its weird to return false if something has been found. 
So you might want to turn your logic around and return true:
```suggestion
    return !!this.flowStructure.configuration?.attributes['xmlns:flow']
```

Reply:
i used the second option where it returns a true when value is found